tests/testthat/mainTests/roundedzero_symbols.csv

```diff
@@ -0,0 +1,376 @@
+time_period,time_identifier,geographic_level,country_code,country_name,region_code,region_name,old_la_code,new_la_code,la_name,school_type,num_schools,enrolments,sess_overall_percent,sess_authorised_percent,sess_unauthorised_percent,enrolments_pa_10_exact,enrolments_pa_10_exact_percent
+201718,Academic year,National,E92000001,England,,,,,,Total,20121,6835059,4.20821,3.01996,1.18824,731047,10.69555
+201718,Academic year,National,E92000001,England,,,,,,State-funded primary,16739,3885774,3.68934,2.69421,0.99512,357742,9.20645
+201718,Academic year,National,E92000001,England,,,,,,State-funded secondary,3382,2949285,4.89899,3.45364,1.44535,373305,12.65747
+201718,Academic year,Regional,E92000001,England,E12000001,North East,,,,State-funded primary and secondary,1033,317616,4.62038,3.12503,1.49536,39300,12.37343
+201718,Academic year,Regional,E92000001,England,E12000001,North East,,,,State-funded primary,858,175079,3.90145,2.7663,1.13516,18073,10.32277
+201718,Academic year,Regional,E92000001,England,E12000001,North East,,,,State-funded secondary,175,142537,5.51279,~,1.94247,21227,14.89227
+201718,Academic year,Regional,E92000001,England,E12000002,North West,,,,State-funded primary and secondary,2905,931181,4.19093,2.89117,1.29976,99903,10.72863
+201718,Academic year,Regional,E92000001,England,E12000002,North West,,,,State-funded primary,2443,531992,3.65441,2.57801,1.0764,49026,9.21555
+201718,Academic year,Regional,E92000001,England,E12000002,North West,,,,State-funded secondary,462,399189,4.91411,3.31328,1.60083,50877,12.74509
+201718,Academic year,Regional,E92000001,England,E12000003,Yorkshire and the Humber,,,,State-funded primary and secondary,2088,693768,4.45385,2.93793,1.51592,82336,11.86794
+201718,Academic year,Regional,E92000001,England,E12000003,Yorkshire and the Humber,,,,State-funded primary,1771,394967,3.85078,2.64836,1.20243,40339,10.21326
+201718,Academic year,Regional,E92000001,England,E12000003,Yorkshire and the Humber,,,,State-funded secondary,317,298801,5.26744,3.32858,1.93886,41997,14.05517
+201718,Academic year,Regional,E92000001,England,E12000004,East Midlands,,,,State-funded primary and secondary,1921,592559,4.15794,3.00161,1.15633,62888,10.61295
+201718,Academic year,Regional,E92000001,England,E12000004,East Midlands,,,,State-funded primary,1633,338065,3.63591,2.65821,0.9777,~,9.17723
+201718,Academic year,Regional,E92000001,England,E12000004,East Midlands,,,,State-funded secondary,288,254494,4.85183,3.45806,1.39377,31863,12.52014
+201718,Academic year,Regional,E92000001,England,E12000005,West Midlands,,,,State-funded primary and secondary,2185,767407,4.20781,2.97631,1.2315,82604,10.76404
```

Comment:
Should drop any rows we don't need so that this file is like 10-15 rows, saves the repo bloating too much as more test data is added